src/current/cockroachcloud/serverless-resource-usage.md

```diff
@@ -95,13 +101,64 @@ Time: 71ms total (execution 20ms / network 50ms)
+
+### Excessive queries
+
+Each query has an associated RU cost, so the total number of queries is an important factor in your consumption.
+
+To diagnose excessive queries, navigate to your cluster's [**Metrics** page]({% link cockroachcloud/metrics-page.md %}) in the {{ site.data.products.cloud }} Console. The **SQL Statements** chart displays the number of queries over time. Look for any spikes or increases in QPS (queries per second) that may correspond to increases in your resource consumption.
+
+Reducing the rate of queries is application-specific and must be achieved at the application level.
+
+### Excessive number of connections
+
+CockroachDB {{ site.data.products.serverless }} consumes minimal resources per connection, so the number of connections is not the most likely cause of increased RU consumption. However, managing your connections is increasingly important for both performance optimization and RU consumption as your application scales up.
+
+To diagnose excessive connections, navigate to your cluster's [**Metrics** page]({% link cockroachcloud/metrics-page.md %}) in the {{ site.data.products.cloud }} Console. The **SQL Connection Attempts** chart displays new SQL connection attempts over time. Maintaining fewer than five active connections is recommended for most workloads.
+
+[Connection pooling](https://www.cockroachlabs.com/docs/stable/connection-pooling) is the recommended way to manage the number of connections for many workloads. To read more about connection pooling, see our [What is Connection Pooling, and Why Should You Care](https://www.cockroachlabs.com/blog/what-is-connection-pooling/) blog post.
+
+### Excessive data egress
+
+The {{ site.data.products.cloud }} Console does not currently provide direct observability of data egress, but you can observe the component of egress that comes from SQL Statements:
+
+ 1. Navigate to the [**Statements** tab]({% link cockroachcloud/statements-page.md %}) of your cluster's **SQL Activity** page in the {{ site.data.products.cloud }} Console.
+1. Sort the queries by the **Bytes Read** column. Most queries should read fewer than 1000 bytes.
+
+Excessive egress can be treated similarly to [expensive queries](#expensive-queries). Reducing the amount of data returned per query is often the best way to decrease egress data. You can also reduce the frequency of [excessive queries](#excessive-queries).
+
+### Database UI tools
```

Comment:
Same comment as before: Include content from Shannon's comment

Database UI tools can also consume a lot of resource units because they can run expensive queries to populate views. In addition, these tools, if left running, will periodically refresh in the background and contribute to RU consumption. 

To determine whether database UI tools are contributing to RU usage, visit the SQL Activity page and look for queries similar to the following.



SELECT count(*) FROM crdb_internal.cluster_sessions

SELECT avg((((statistics->'')->'')->'_')::INT8) AS meanrunlatency
FROM crdb_internal.statement_statistics AS ciss
WHERE ciss.aggregated_ts::DATE = current_date()
You might also see multiple open connections to your cluster persisting regardless of workload. This can indicate that multiple team members are using a database UI tool and leaving the application running.

We might want to recommend a means of determining whether database UI tools are the problem such as ensuring all sessions have been disconnected (possibly using CANCEL SESSION).